docs/misc/constants.rst

```diff
@@ -10,4 +10,4 @@ The ``OidcConstants`` class has all the values for grant types, parameter names,
 JWT Claim Types
 ---------------
 The ``JwtClaimTypes`` class has all standard claim types found in the OpenID Connect, JWT and OAuth 2.0 specs -
-many of them are also aggregated at `IANA <https://www.iana.org/assignments/jwt/jwt.xhtml>`_.
\ No newline at end of file
+many of them are also aggregated at `IANA <https://www.iana.org/assignments/jwt/jwt.xhtml>`_.
```

Comment:
GitHub's web editor insists on adding a newline at the end, I can't seem to undo it.